unity/Assets/Scripts/LocobotFPSAgentController.cs

```diff
@@ -52,9 +52,9 @@ public class LocobotFPSAgentController : BaseFPSAgentController {
             standingLocalCameraPosition = m_Camera.transform.localPosition;
             crouchingLocalCameraPosition = m_Camera.transform.localPosition + new Vector3(0, -0.2206f, 0);// smaller y offset if Bot
 
-            // limit camera from looking too far down
-            this.maxDownwardLookAngle = 30f;
-            this.maxUpwardLookAngle = 29f;
+            // limit camera from looking too far down/up
+            this.maxDownwardLookAngle = 60f + 1e-6f; // Adding 1e-6f in case of floating point issues.
```

Comment:
the epsilon should not be needed. The helper function `CheckForUpDownAngleLimit` that parses these max angles accounts for the epsilon already

Reply:
Great, done for both.